package.json

```
@@ -37,14 +37,12 @@
     "url": "git://github.com/krakenjs/belter.git"
   },
   "keywords": [
-    "template"
-  ],
-  "licenses": [
-    {
-      "type": "Apache 2.0",
-      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
-    }
+    "template",
```

My guess is "template" was originally pasted from a package.json template 😄  Probably safe to remove
```suggestion
    "template",
```